src/network/serverpackethandler.cpp

```diff
@@ -716,6 +716,18 @@ void Server::handleCommand_ClientReady(NetworkPacket* pkt)
 			peer_id, major_ver, minor_ver, patch_ver,
 			full_ver);
 
+	std::vector<std::string> players = m_clients.getPlayerNames();
+	std::vector<std::string>::iterator it = players.begin();
+	for(; it != players.end(); ++it) {
```

Comment:
C++11 permits to replace this with:

for (const std::string &player: m_clients.getPlayerNames()) {
}